### DISCUSSION TITLE
Slurry/Manure

### DISCUSSION THREAD

**Discussion by MikeDK2004** [Courseplay/Courseplay_FS22#1753] [General]
When trying to fertilize either by Slurry or Manure the worker just stops, and says "Tool empty"
Any idea how to fix this?

**Comment by schwiti6190** [comment]
@MikeDK2004 Do you have the game setting to collect from the cow/pig farms active under game settings, as their is a giants bug, where the helpers stop if the manure silos are empty even if their is something left in the manure/slurry spreader? :)

**Reply by MikeDK2004** [reply]
Yeah, i wanted to empty my cow barn of slurry?
So ur saying that i have to set up a AutoDrive course for my slurry spreader?

**Reply by Tensuko** [reply]
AutoDrive will transport, Courseplay will spray on the field.
Its workimg and I am sure there are tutorials for it on youtube.

CP side: Have a course rdy on the field for your slurry sprayer.

**Reply by schwiti6190** [reply]
@MikeDK2004 I've tested the setting for slurry from cow barns and it seems that the giants helper is also not working with it. This is a giants bug, as we depend on their code. If you like, you can create a bug ticket in their bug tracker :)
